### PR TITLE
Fix database wait logic to not fail if db hasn't been created

### PR DESCRIPTION
**Pull request by philip-cc**
### Description
Previously, if the Postgres database was alive, but the `authn` database had not yet been created, our ping test would pass, only for the schema command to fail. This PR fixes that. 

Also updates the version of pgx, and adds a wait to the E2E test script. 

### Parent Issue
None

### Related Pull Requests
None

### Comments for the Reviewers
You can test this by adding a `sleep` to `cockroachdb-init` in docker compose. 

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [x] Bugfix
- [ ] Updated documentation
- [ ] Infrastructure changes (Terraform, GCP, K8s, other)
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [x] I have pulled in the latest changes from master.
- [x] I have run `make lint`.
- [x] I have successfully run `make docker-up; make tests`.
- [ ] I have updated relevant workflows and deployment tools.
- [ ] I have updated/added relevant documentation (readme, doc comments, etc).
- [ ] I have added the license to new source code files.
- [x] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: encryption-service/impl/authstorage/authstorage.go [hunk 1]
```diff
@@ -143,17 +143,14 @@ func (store *AuthStore) ImportSchema(ctx context.Context, schemaFile string) err
 	// Wait for DB to be up
 	// TODO: this is not ideal
 	for i := 0; i < 120; i++ {
-		// TODO: replace builtin pgxpool once this is released:
-		// https://github.com/jackc/pgx/commit/aa8604b5c22989167e7158ecb1f6e7b8ddfebf04
-		_, err := store.Pool.Exec(ctx, ";")
+		_, err = store.Pool.Exec(ctx, string(schemaData))
```
Comment (bruno-cc) [inline encryption-service/impl/authstorage/authstorage.go:146]: I would add a comment what we do here and why it's not a ping

File: encryption-service/impl/authstorage/authstorage.go [hunk 2]
```diff
@@ -143,17 +143,14 @@ func (store *AuthStore) ImportSchema(ctx context.Context, schemaFile string) err
...
 		if err == nil {
 			break
 		}
 
 		log.Debugf(ctx, "Auth Storage ping failed (retrying ...) - %v", err)
```
Comment (bruno-cc) [inline encryption-service/impl/authstorage/authstorage.go:151]: it's a setup instead of ping now

**Review by bruno-cc — DISMISSED** [review]
See minor comments about comments

File: encryption-service/impl/authstorage/authstorage.go (hunk 1 above)
Comment (philip-cc) [reply encryption-service/impl/authstorage/authstorage.go:147]: Fixed

File: encryption-service/impl/authstorage/authstorage.go (hunk 2 above)
Comment (philip-cc) [reply encryption-service/impl/authstorage/authstorage.go]: Fixed

**Review by bruno-cc — APPROVED** [review]

**Pull request merged**